data/category-porn

```diff
@@ -5207,6 +5208,7 @@ regexp:(^|\.)[1-9]+zipai\.(com|net)$
 regexp:(^|\.)[1-9]00caocao\.com$
 regexp:(^|\.)[3-9]0000sao\.com$
 regexp:(^|\.)[a-z][1-9][0-9][a-z]\.com$
+regexp:(^|\.).*fulibl.*\.(cc|com|org|net|xyz)$
```

Comment:
it does look right to me, what are the domains you are trying to match against?

Reply:
```
bfulibl.net
fulibl.cc
fulibl.com
fulibl.net
fulibl.org
fulibl.xyz
ifulibl.net
```

Sometimes, it will add an alphabet before `fulibl`. Actually, I wanna use the syntax `keyword:`, but I am afraid it will match the domain which should not be matched.

Reply:
> ```
> bfulibl.net
> fulibl.cc
> fulibl.com
> fulibl.net
> fulibl.org
> fulibl.xyz
> ifulibl.net
> ```
> 
> Sometimes, it will add an alphabet before `fulibl`. Actually, I wanna use the syntax `keyword:`, but I am afraid it will match the domain which should not be matched. And no one uses it in this repository, so I choose the `regexp:` to realize it.

---

In that case, I will suggest the following regexp:
```
regexp:^([a-zA-Z0-9]*)fulibl\.(cc|com|net|org|xyz)$
```

it makes sense to cover some domains that are not being used for now, but the previous one had gone too far...